Archive/Tar.php

```diff
@@ -244,7 +244,7 @@ public function __construct($p_tarname, $p_compress = null, $buffer_length = 512
                 "a8checksum/a1typeflag/a100link/a6magic/a2version/" .
                 "a32uname/a32gname/a8devmajor/a8devminor/a131prefix";
         } else {
-            $this->_fmt = "Z100filename/Z8mode/Z8uid/Z8gid/Z12size/Z12mtime/" .
+            $this->_fmt = "Z100filename/Z8mode/Z8uid/Z8gid/a12size/Z12mtime/" .
```

Comment:
What do you think about adding a version check that still uses Z for 5.4 ? Maybe that would resolve the build test failure, and allow us to not yet be forced to drop older support.